Example/PrimerSDK.xcodeproj/project.pbxproj

```diff
@@ -885,6 +885,7 @@
 				INFOPLIST_FILE = PrimerSDK/Info.plist;
 				IPHONEOS_DEPLOYMENT_TARGET = 12.1;
 				LD_RUNPATH_SEARCH_PATHS = "$(inherited) @executable_path/Frameworks";
+				MARKETING_VERSION = "1.0";
```

Comment:
@xevious78 don't you want the marketing version to take the value of the SDK version?

Reply:
Good catch, not sure I want this to be part of the PR.